### PR TITLE
STS check latest transfer operation sample

### PR DESCRIPTION
**Pull request by JesseLovelace**
Adds a new sample for checking the latest transfer operation in Storage Transfer Service

### CODE REVIEW

File: storage/storage-transfer/src/test/java/com/google/cloud/storage/storagetransfer/samples/test/CheckLatestTransferOperationTest.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    final ByteArrayOutputStream sampleOutputCapture = new ByteArrayOutputStream();
+    System.setOut(new PrintStream(sampleOutputCapture));
+
+    CheckLatestTransferOperation.CheckLatestTransferOperation(PROJECT_ID, response.getName());
+
+    String sampleOutput = sampleOutputCapture.toString();
+    System.setOut(standardOut);
+    assertThat(sampleOutput.contains(response.getName()));
+  }
+}
```
Comment (frankyn) [inline storage/storage-transfer/src/test/java/com/google/cloud/storage/storagetransfer/samples/test/CheckLatestTransferOperationTest.java:57]: Add new lines to end of both files.

File: storage/storage-transfer/src/main/java/com/google/cloud/storage/storagetransfer/samples/CheckLatestTransferOperation.java [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+public class CheckLatestTransferOperation {
+
+  // Gets the requested transfer job and checks its latest operation
+  public static void CheckLatestTransferOperation(String projectId, String jobName)
+      throws IOException {
+    // Your Google Cloud Project ID
+    // String projectId = "your-project-id";
+
+    // The name of the job to check
+    // String jobName = "my-job-name";
```
Comment (frankyn) [inline storage/storage-transfer/src/main/java/com/google/cloud/storage/storagetransfer/samples/CheckLatestTransferOperation.java:39]: Is there an example you could use in place of `my-job-name` or is this the format it uses?

**Review by frankyn — CHANGES_REQUESTED** [review]

File: storage/storage-transfer/src/main/java/com/google/cloud/storage/storagetransfer/samples/CheckLatestTransferOperation.java (hunk 2 above)
Comment (JesseLovelace) [reply storage/storage-transfer/src/main/java/com/google/cloud/storage/storagetransfer/samples/CheckLatestTransferOperation.java:39]: I'll update it with a better example

**Comment by eaball35** [comment]
Screen Shot 2020-11-30 at 8 39 23 AM
Screen Shot 2020-11-30 at 8 40 05 AM
Screen Shot 2020-11-30 at 8 40 25 AM

**Review by eaball35 — CHANGES_REQUESTED** [review]
please address comments

**Review by frankyn — APPROVED** [review]

**Review by eaball35 — APPROVED** [review]

**Pull request merged**